tools/ansible-runner/Dockerfile

```diff
@@ -6,7 +6,7 @@ LABEL maintainer="OpenEBS"
 RUN rm -rf /var/lib/apt/lists/* && \
     apt-get clean && \
     apt-get update --fix-missing || true && \
-    apt-get install -y python-minimal python-pip \
+    apt-get install -y jq python-minimal python-pip \
```

Comment:
We could go ahead with this - just wanted to see if we can use kubectl natively (there are some thoughts around bundling lib/tests so they can downloaded/runnable in/outside containers - i.e., wherever kubeconfig is present)

Reply:
removed the dependency on jq